pages/api/[entity_type_slug].js

```diff
@@ -55,6 +55,7 @@ async function get(req, res) {
         queries
     );
     const rawEntityType = await EntityType.find({ slug: entity_type_slug });
+    const isSingleType = rawEntityType.map(x => x.entity_type_variant === 'singleton')[0];
```

Comment:
We can save looping through the whole results by directly peeking at the value of the first entry.
```suggestion
    const isSingleType = (rawEntityType[0].entity_type_variant === 'singleton');
```